docs/src/components/ColorSwatchFamilyList/ColorSwatchFamilyList.component.tsx

```diff
@@ -0,0 +1,82 @@
+// REACT
+import React, { useState } from 'react';
+// VENDOR
+import styled from 'styled-components';
+import classNames from 'classnames';
+import { ColorSwatchFamily } from '../ColorSwatchFamily';
+// THEME
+import { colors, sizes } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchFamilyListProps {
+  className?: string;
+}
+
+
+// STYLES
+const StyledColorSwatchFamilyList = styled.div`
+
+`;
+
+
+// JSX MARKUP
+export const ColorSwatchFamilyList = ({
+  className
+}: ColorSwatchFamilyListProps): React.ReactElement<any> => {
+
+  const [copyChoice, setCopyChoice] = useState('hexCode');
+
+  return (
+    <StyledColorSwatchFamilyList
+      className={classNames(className)}
+    >
+
+      <p> Each color swatch has 4 pills representing the color contrast when used on these colors:</p>
+      <ul>
+        <li><code>#FFFFFF</code></li>
+        <li><code>#FAFAFA</code></li>
+        <li><code>#F1F1F1</code></li>
+        <li><code>#000000</code></li>
+      </ul>
+      <p>Click any swatch to copy either the hex code or the variable name, based on your choice here:</p>
+
+      <p>
+        <input
+          type="radio"
+          id="copyChoiceHexCode"
+          name="copyChoice"
+          value="hexCode"
+          checked={copyChoice == 'hexCode'}
+          onClick={ () => setCopyChoice('hexCode')}
+        >
+        </input>
+        <label for="copyChoiceHexCode"> Hex Code</label>
```

Comment:
React is giving a warning that the label doesn't work. It should be `htmlFor=`. But I prefer wrapping the element with the label over using `for`/`id`s anyway:
```html
        <label>
          <input type="radio" name="copyChoice" value="hexCode"
          /> Hex Code
        </label>
```
and
```jsx
        <label>
          <input type="radio" name="copyChoice" value="variableName"
          /> Variable Name
        </label>
```

Reply:
The wrapping syntax here I can go either way on. Just an FYI in a production app you'd still need the `htmlFor` as well as `aria` tags to promote a11y, but this is the docs site, so I'm not going to police that here. Further, I'm working on the `Input` for Anchor right now so eventually you'll get some of this for free.

Reply:
I'm going to look into the `htmlFor` and `aria` because I'd like to practice doing it the right way.

Reply:
Found this: https://a11y-style-guide.com/style-guide/section-forms.html#kssref-forms-radio-buttons